pkg/cli/cmd/list/list.go

```diff
@@ -0,0 +1,130 @@
+package list
+
+import (
+	"fmt"
+	"log"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/fatih/color"
+	"github.com/openshift-pipelines/manual-approval-gate/pkg/actions"
+	"github.com/openshift-pipelines/manual-approval-gate/pkg/apis/approvaltask/v1alpha1"
+	cli "github.com/openshift-pipelines/manual-approval-gate/pkg/cli"
+	"github.com/openshift-pipelines/manual-approval-gate/pkg/cli/flags"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type ListOptions struct {
+	AllNamespaces bool
+}
+
+var (
+	taskGroupResource = schema.GroupVersionResource{Group: "openshift-pipelines.org", Resource: "approvaltasks"}
+)
+
+var ConditionColor = map[string]color.Attribute{
+	"Rejected": color.FgHiRed,
+	"Approved": color.FgHiGreen,
+	"Pending":  color.FgHiBlue,
+}
+
+const listTemplate = `{{- $at := len .ApprovalTasks.Items }}{{ if eq $at 0 -}}
```

Comment:
This table makes not much sense for me. Too much information repeated. We can go for something like

```
NAME	   Approvals 	       STATUS
foo 	           2/3            	       Wait
bar 	           4/4            	       Approved
```

Reply:
As we are supporting only single reject, showing a column does not make any sense

Reply:
@piyush-garg I am fine with it as long as its clear that in the example of 0/3 and status as "rejected" someone from the approval list has rejected it.